### DISCUSSION TITLE
More Datacap ownership management

### DISCUSSION THREAD

**Discussion by jennijuju** [filecoin-project/notary-governance#318] [Ideas]
A FIP discussion is created [here](https://github.com/filecoin-project/FIPs/issues/204) for adding `TransferDatacap` and `RemoveVerifiedClient`/`RevokeDatacap` potentially. Please head over to the FIP issue for more details.

**Comment by dkkapur** [comment]
@jennijuju - nice, thanks for sharing! Just in time for the governance call in ~30min, will flag this!
